src/app/modules/main/communities/tokens/module.nim

```diff
@@ -161,12 +175,31 @@ method deployCollectibles*(self: Module, communityId: string, fromAddress: strin
   self.tempDeploymentParams.transferable = transferable
   self.tempDeploymentParams.remoteSelfDestruct = selfDestruct
   self.tempDeploymentParams.tokenUri = utl.changeCommunityKeyCompression(communityId) & "/"
+  self.tempDeploymentParams.ownerTokenAddress = ownerToken.address
+  self.tempDeploymentParams.masterTokenAddress = masterToken.address
   self.tempTokenMetadata.tokenType = TokenType.ERC721
   self.tempTokenMetadata.description = description
   self.tempTokenImageCropInfoJson = imageCropInfoJson
   self.tempContractAction = ContractAction.Deploy
   self.authenticate()
 
+method deployOwnerToken*(self: Module, communityId: string, fromAddress: string, ownerName: string, ownerSymbol: string, ownerDescription: string,
+                        masterName: string, masterSymbol: string, masterDescription: string, chainId: int, imageCropInfoJson: string) =
+  self.tempAddressFrom = fromAddress
```

Comment:
It might be worth it to check if the contracts are already deployed here, like you did above. So that we don't deploy it twice (at least until we support it for multiple chains)